x-pack/plugins/monitoring/kibana.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "monitoring",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
+  "configPath": ["monitoring"],
+  "requiredPlugins": ["usageCollection", "licensing", "features", "alerting", "actions", "infra"],
```

Comment:
I was impressed that alerting and actions are optional dependencies https://github.com/elastic/kibana/pull/56675/files#diff-da1f3c47dadcfeda29778aa7508e29d0R20

Reply:
That's true, they are not required right now